docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
The anchors in the first format don't work when displayed on Github but they do work after generating the docs.  Is the second form also compatible with Sphinx?